src/features/commands/user.js

```diff
@@ -196,6 +196,8 @@ module.exports = {
             case 'warnings':
                 switch (interaction.options._hoistedOptions[0]?.name) {
                     case 'user':
```

Comment:
```suggestion
                    if (['user', 'user-id', 'user-name'].includes(interaction.options._hoistedOptions[0]?.name) {
                    } else {
                    }
```
Could be a potentially cleaner solution than a switch block.